### PR TITLE
Add hash.dev hook service blog post + crosslink to hash.ai blog

### PR DESCRIPTION
**Pull request by CiaranMn**
## 🌟 What is the purpose of this PR?

Adds a blog post announcing the hook service. This required introducing support for multiple authors in the blog page.

It also adds a cross-link from the hash.dev/blog to hash.ai/blog. In the process of doing this I noticed that the links marked with an 'open in new' icon were not opening in a new tab, so I fixed those too.

## 🔗 Related links

- [Asana task for blogpost](https://app.asana.com/0/1200211978612931/1203123950268254) _(internal)_
- [Asana task for crosslink](https://app.asana.com/0/1200036380334625/1202440221033825) _(internal)_

## 🚫 Blocked by

- [x] Cover image for blog post

## ❓ How to test this?

Wait for the `hashdev` deployment (in checks), and then visit it

## 📹 Demo
Screenshot 2022-10-21 at 18 39 10
Screenshot 2022-10-21 at 18 39 20

### CODE REVIEW

File: sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+Block-based applications are no different. Think WordPress, Contentful, Notion.
+
+At HASH, we’re helping develop a new open standard: the Block Protocol. Its goal is to allow any [block](https://blockprotocol.org/docs#what-is-a-block) to be used in any application, with zero configuration
+
+It does this by standardizing how blocks and applications communicate, and what messages they exchange.
+
+Because rich text editing is a core part of many modern web applications, we knew the Block Protocol had to support it.
+
+We had to decide how blocks made by different people could provide a consistent text editing experience _within_ an application, while allowing for variations in the experience **\***across**\*** applications – without the block knowing anything about the application it’s being used in.
```
Comment (nonparibus) [inline sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx:59]: This is causing some weirdness

Screenshot of rendered site being weird

File: sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
+---
+authors:
+  - name: Ciaran Morinan
+    jobTitle: Head of Engineering, HASH
+    photo: blog/authors/ciaran-morinan.jpg
+  - name: Nate Higgins
+    jobTitle: Senior Platform Engineer, HASH
+    photo: blog/authors/nate-higgins.jpg
+postPhoto: blog/0004_announcing-error-stack/ErrorStack_Cover.jpg
+title: "Announcing the Block Protocol hook service"
```
Comment (nonparibus) [inline sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx:10]:
```suggestion
title: "The Block Protocol hook service"
```

File: sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+## Complexities underlined
+
+There are two parties to the Block Protocol – the block, and the application using it.
+
+To display some formatted text on a web page, _one_ of these parties has to take a representation of the text and render it.
+
+### Representing text
+
+We can’t expect blocks to take any old text and understand it. The same words with the same formatting in different applications can look identical on the screen, but vary wildly in how they look behind the scenes. Text might be stored as Markdown, HTML, or any of a [number](https://developers.notion.com/reference/rich-text) [of](https://prosemirror.net/docs/ref/#model.Node) [other](https://www.contentful.com/developers/docs/concepts/rich-text/#rich-text-on-the-api-response) [approaches](https://en.wikipedia.org/wiki/OpenDocument) to storing a series of characters with formatting.
```
Comment (nonparibus) [inline sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx:69]: Let's sub out the Contentful/Notion links to examples of open-source libraries. e.g. Slate, Quill, TinyMCE, etc. Whichever seem to be most different/contrasting in approach.

**Review by nonparibus — CHANGES_REQUESTED** [review]
Couple of minor things. I'll add a cover image shortly.

File: sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx (hunk 3 above)
Comment (CiaranMn) [reply sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx]: Done in https://github.com/hashintel/hash/pull/1241/commits/a1cab063dfff276d856ab957c2110263e5b9ea60

File: sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx (hunk 1 above)
Comment (CiaranMn) [reply sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx]: https://github.com/hashintel/hash/pull/1241/commits/a1cab063dfff276d856ab957c2110263e5b9ea60

**Review by nonparibus — APPROVED** [review]

**Pull request merged**